packages/web-components/src/display/natValue/parseAsNat.ts

```diff
@@ -17,11 +16,10 @@ import { roundToDecimalPlaces } from './helpers/roundToDecimalPlaces.js';
  * still 300n cents with decimalPlaces =2 because the thousandths place is dropped.
  *
  * In the future, we may add a parameter to change the rounding rules.
- * @param {string} str
- * @param {number} decimalPlaces
- * @returns {bigint}
+ * @param str
+ * @param decimalPlaces
```

Comment:
Can we drop these explicit `@param`s in favor of the typed params, and still keep the rest of the doc comment?

Reply:
Not with the current lint rules (discussed above so I'm resolving this one)